### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by thanhchuc26**
https://edu-redmine.sun-asterisk.vn/issues/37281

### CODE REVIEW

**Comment by thanhchuc26** [comment]
ready

**Comment by thanhchuc26** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>DnOe44RubyFoodAndDrink</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb]: Nháy đôi anh bạn

**Comment by thanhchuc26** [comment]
ready

**Pull request merged**